### PR TITLE
fix: #365 Remove Unused Endponts.

### PR DESCRIPTION
**Pull request by ianliuwk1019**
- Backend: Remove unused endpoints for router_user and router_role.
- Frontend: remove unused FAMUsersApi and FAMRolesApi from ApiServiceFactory.
- Bump openapitools/openapi-generator-cli to v7.0.1
- Backend: FastAPI temporary adjustment for swagger generation with app.openapi_version = "3.0.3" and document openapi spec issue causing generated code not working.
- "gen" folder, is generated.
- Manually fixed openapi.json regarding to swagger spec type incorrect on "some" endpoints but currently has no proper solution after investigation.

### CODE REVIEW

File: client-code-gen/gen/api/famapplications-api.ts [hunk 1]
```diff
@@ -71,11 +72,11 @@ export const FAMApplicationsApiAxiosParamCreator = function (configuration?: Con
         /**
          * gets the roles associated with an application  :param application_id: application id :param db: database session, defaults to Depends(database.get_db)
          * @summary Get Fam Application Roles
-         * @param {number} applicationId 
+         * @param {any} applicationId
```
Comment (MCatherine1994) [inline client-code-gen/gen/api/famapplications-api.ts:75]: Hi Ian, why we change the type for applciationId from number to any?

**Comment by MCatherine1994** [comment]
The rest looks good to me, thanks so much!!

File: client-code-gen/gen/api/famapplications-api.ts (hunk 1 above)
Comment (ianliuwk1019) [reply client-code-gen/gen/api/famapplications-api.ts:75]: Hi Catherine, the code is generated based on backend swagger spec, it is strange why it is changed. Let me try to figure it out.

Comment (ianliuwk1019) [reply client-code-gen/gen/api/famapplications-api.ts]: Hi Catherine, thanks for spotting this. Indeed, it is a strange case that some does get generated correctly but few don't from backend FastAPI.
I took some time to downgrade and upgrade but none can solve this issue. For now, as discussed, I manually added the missing schema type into problem endpoints and add section for note to the README.
Please have a look again. Thank you!!

**Review by MCatherine1994 — APPROVED** [review]

**Pull request merged**